app/controllers/application_controller.rb

```diff
@@ -1598,7 +1598,7 @@ def get_view(db, options = {})
       :page                      => options[:all_pages] ? 1 : @current_page,
       :per_page                  => options[:all_pages] ? ONE_MILLION : @items_per_page,
       :where_clause              => get_view_where_clause(options[:where_clause]),
-      :named_scope               => options[:named_scope],
+      :named_scope               => options[:named_scope] || (:without_volume_templates if db == "MiqTemplate"),
```

Comment:
We cannot do this :-(

We are trying to make the code here generic, remove exceptions, cleanup so that we can move towards using some API from the GTL component.

This goes the other way round and adds an exception to a place where noone is going to have any idea what this is about the momen this is merged :-(

Reply:
That's why I wrote in the description that I knew it was doing it wrong. ;) Where's the right place to add the named scope? I thought the report would be the right place, but it's not supported there.

Reply:
It depends. Can you show me where the list is being displayed? Is there a `process_show_list` call there?

Reply:
The image list is usually accessed through `http://localhost:3000/ems_cloud/<id>?display=images`, but it can also be accessed through the explorer view. 
![screenshot from 2017-10-03 15-54-33](https://user-images.githubusercontent.com/628956/31145968-31c5bcd0-a853-11e7-97d3-3f1771c28af1.png)

Reply:
Yes, so ems_cloud includes EmsCommon and that includes GenericShow. Generic show has `display_images`.

So you might need to change this so that the `display_images` is not shared. If is some contexts it has to show different stuff.

Or you might need to have a new `display_<something>` method for your case and link to that new display mode.

If you want to limit what is displayed in all the cases then modify the shared `display_images` method.

There might be a problem with propagating some attributes from the `display_*` to where the  `/report_data` request is handled. This is currently being worked on. If that would be the case, then, please, ping me and @karelhala .

Reply:
I'm not certain exactly how the page is rendered after `display_images` is called, but it seems to be rendered twice. I can see two calls to `get_view` pass by in my logging. Once, `get_view` is called for `ManageIQ::Providers::CloudManager::Template` with the parameters passed from `display_images` and the correct named_scope, but it seems to then be overwritten by another call to `get_view` for `MiqTemplate`. Perhaps it is related to the problem you mentioned above, but it seems strange to me that the view would be rendered and then rerendered with a more generic call.

Reply:
`get_view` is called twice. Once when the partial is rendered and once for every request to /report_data. That is an issue we are currently working on.

The first call has to be removed. We need to fix things to have the scope bubble through the call.

Ping @karelhala, we need to resolve this.

Reply:
@martinpovolny @karelhala any updates on this? thanks! :)